backend/route/contributors.route.js

```diff
@@ -0,0 +1,41 @@
+const Boom = require('@hapi/boom');
+const axios = require('axios');
+const linkedIn = require('../shared/contributors')
+
+const getLinkedInUrl = username => {
+    return linkedIn[username]
+}
+
+// Get all contributors
+module.exports = (server) => {
+    server.route({
+        method: 'GET',
+        path: '/api/v1/contributors',
+        handler: async (request, h) => {
+            try {
+                const { data } = await axios.get(
+                    `https://api.github.com/repos/MintbeanHackathons/MintbeanPlatform/contributors`
+                );
+                const contributors = data;
+                for (const contributor of contributors) {
+                    const { data } = await axios.get(contributor.url);
+                    contributor.name = data.name;
+                    contributor.portfolio = addHttpPrefix(data.blog);
+                    contributor.linkedIn = getLinkedInUrl(contributor.login);
+                }
+                return contributors;
+            } catch (e) {
+                throw Boom.badRequest('Was not able to retrieve contributors.')
+            }
+        }
+    });
+};
+
+const addHttpPrefix = url => {
+    if (
+        url.toLowerCase().startsWith('http://') || url.toLowerCase().startsWith('https://')
+    ) {
+        return 'http://' + url
+    }
+    return url
+}
```

Comment:
Just saw this. I'm taking a look now...

Reply:
Does this need to return `null` if `url` is empty? Since `url` is falsy when it's set to either an empty string or `null`, I'm pretty sure the behavior on the front end will be the same.

Reply:
Good point. Updating it now.

Reply:
@carloshdelreal, I did it a little differently, but it should work just the same.